pkg/azure/aks/pricing_map_test.go

```diff
@@ -0,0 +1,7 @@
+package aks
+
+import "testing"
+
+func TestMapCreation(t *testing.T) {
+	t.Skip()
+}
```

Comment:
Seems rather thin 😉 We can pair up later today on how to create mock objects so you can do more robust tests for the `PopulatePriceStore` calls.

Can you add tests for the other methods that do not make any api requests?

Reply:
added